endpoints.yml

```diff
@@ -0,0 +1,2100 @@
+---
+endpoints:
+  core:home:
+    - /core/
+  core:about:
+    - /core/about/
+  core:search:
+    - /core/search/
+  core:graphql:
+    - /core/graphql/
+  core:media_failure:
+    - /core/media-failure/
+  core:db_file_storage.download_file:
+    - /core/files/download/
+  core:worker_status:
+    - /core/worker-status/
+  core:render_jinja_template:
+    - /core/render-jinja-template/
+  core:theme_preview:
+    - /core/theme-preview/
+  core:metrics:
+    - /core/metrics/
+  circuits:circuit_terminations_swap:
+    - /circuits/circuits/<uuid:pk>/terminations/swap/
```

Comment:
Looking at the original yaml file, we should probably for now only include GET-capable endpoints, not those that actually modify the data and/or expect a POST request.